std/mem.zig

```diff
@@ -84,6 +84,36 @@ pub const Allocator = struct {
     }
 };
 
+pub var c_allocator = Allocator {
+    .allocFn = cAlloc,
+    .reallocFn = cRealloc,
+    .freeFn = cFree,
+};
+
+const c = @cImport(@cInclude("stdlib.h"));
```

Comment:
I think in this case we should `@import("c/index.zig")` (see https://github.com/zig-lang/zig/blob/master/std/c/index.zig) and add `malloc`, `realloc`, and `free`. This avoids the overhead of parsing .h files, and as you pointed out, can still work when no .h files are present.

We can let there be linker errors if the programmer fails to link libc.